addins/upgrade/upgrade_shared.html

```diff
@@ -26,6 +26,21 @@
 			function getSharedValueIntoUI() {
 				document.getElementById("getValue").innerText = g_sharedState.value;
 			}
+
+			function setStartupStateValueFromUI() {
+				const startupState = document.getElementById("setStartupStateValue").value;
```

Comment:
Didn't we have problems with 'const' before? Please use the good old 'var' to keep IE happy.